engine/Assets/Scripts/UI/Dynamic/MainHUD.cs

```diff
@@ -445,6 +445,11 @@ public enum DrawerPosition {
             }
         }, drawerPosition: DrawerPosition.Bottom);
 
+#if UNITY_EDITOR
+        AddItemToDrawer("Skip to End", b => MatchMode.MatchTime = MatchMode.MatchTime > 10 ? 10 : MatchMode.MatchTime,
+            drawerPosition: DrawerPosition.Bottom);
+#endif
+
```

Comment:
Is it not possible to do this outside of unity? Or is it just for us during testing?

Reply:
Tis just for us. Users can go to practice mode if they don't want the timer.